firmware/application/apps/lge_app.hpp

```diff
@@ -79,28 +84,28 @@ class LGEView : public View {
 	
 	Labels labels {
 		//{ { 7 * 8, 1 * 8 }, "NO FUN ALLOWED !", Color::red() },
-		{ { 1 * 8, 1 * 8 }, "Trame:", Color::light_grey() },
-		{ { 1 * 8, 3 * 8 }, "Salle:", Color::light_grey() },
-		{ { 14 * 8, 3 * 8 }, "Texte:", Color::light_grey() },
-		{ { 0 * 8, 5 * 8 }, "Equipe:", Color::light_grey() },
-		{ { 0 * 8, 7 * 8 }, "Joueur:", Color::light_grey() },
-		{ { 0 * 8, 10 * 8 }, "Collier:", Color::light_grey() },
+		{ { 1 * 8, 1 * 8 }, "Frame:", Color::light_grey() },
+		{ { 2 * 8, 3 * 8 }, "Room:", Color::light_grey() },
+		{ { 14 * 8, 3 * 8 }, "Text:", Color::light_grey() },
+		{ { 1 * 8, 5 * 8 }, "Group:", Color::light_grey() },
+		{ { 0 * 8, 7 * 8 }, "Player:", Color::light_grey() },
+		{ { 0 * 8, 10 * 8 }, "Vest:", Color::light_grey() },
```

Comment:
'Collier' is the French for Collar or Necklace. I don't know if 'Vest' which is kind of a synonym for jacket is applying here

Reply:
I guess vest should be ok. Since it's really a vest you put on when playing the game.